### PR TITLE
Rebranded for witinyourmacros and removed barebones branding

### PR DESCRIPTION
**Pull request by kevinsa**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -19,7 +20,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Bare Bones</title>
+    <title>withinyourmacros</title>
```
Comment (joseph4tw) [inline client/public/index.html]: Maybe we should have it `Within Your Macros` to make it more professional?

**Comment by kevinsa** [comment]
[Re-brand the application UI](https://trello.com/c/ZifWChOJ/11-re-brand-the-application-ui)

File: client/public/index.html (hunk 1 above)
Comment (kevinsa) [reply client/public/index.html]: I agree, we'll need to revisit this once we have some more branding identity.  We could break up the works with spaces, or we could leave them together and use bold/fonts to break them up, etc...  Ideally the title should be more like "daily marconutrient tracking at it's finest | withinmymacros".  We'll have to decide on something when we are further along.

Comment (joseph4tw) [reply client/public/index.html]: Cool, sounds good

**Pull request merged**